cobrawap/pipeline/stage02_processing/scripts/logMUA_estimation.py

```diff
@@ -175,14 +175,13 @@ def plot_logMUA_estimation(asig, logMUA_asig, highpass_frequency, lowpass_freque
     if args.plot_channels is not None:
         for channel in args.plot_channels:
             plot_logMUA_estimation(asig=block.segments[0].analogsignals[0],
-                                logMUA_asig=asig,
-                                highpass_frequency=args.highpass_frequency*pq.Hz,
-                                lowpass_frequency=args.lowpass_frequency*pq.Hz,
-                                t_start=args.plot_tstart,
-                                t_stop=args.plot_tstop,
-                                channel=channel)
-            output_path = args.img_dir \
-                        / args.img_name.replace('_channel0', f'_channel{channel}')
+                                   logMUA_asig=asig,
+                                   highpass_frequency=args.highpass_frequency*pq.Hz,
+                                   lowpass_frequency=args.lowpass_frequency*pq.Hz,
+                                   t_start=args.plot_tstart,
+                                   t_stop=args.plot_tstop,
+                                   channel=channel)
+            output_path = args.img_dir / args.img_name.replace('_channel0', f'_channel{channel}')
```

Comment:
This line becomes too long. Better to keep it as two lines.

Reply:
Ok, fixed.